spec/index.bs

```diff
@@ -7350,10 +7350,17 @@ of the given {{GPUSwapChainDescriptor}}.{{GPUSwapChainDescriptor/device}}, initi
 {{GPUTextureFormat/"rgba8unorm"}}, {{GPUTextureFormat/"rgba8unorm-srgb"}}&raquo;.
 
 <script type=idl>
+
+enum GPUSwapChainAlphaMode {
+    "opaque",
+    "premultiplied",
+};
+
 dictionary GPUSwapChainDescriptor : GPUObjectDescriptorBase {
     required GPUDevice device;
     required GPUTextureFormat format;
     GPUTextureUsageFlags usage = 0x10;  // GPUTextureUsage.RENDER_ATTACHMENT
+    GPUSwapChainAlphaMode alphaMode = "opaque";
```

Comment:
nit: should we call this a "composition mode" or something since it doesn't affect alpha as seen on the WebGPU side?

Reply:
@jdashg please rename this to `GPUCompositeAlphaMode` before landing

Reply:
And rename the dictionary member to `compositeAlphaMode` (or something along those lines) as well.

Reply:
Updated to GPUCanvasCompositingAlphaMode and compositingAlphaMode. Decided not to use "composite" because in this position it looks like we're using it as an adjective.